traffic-generator/openshift/traffic-generator.yml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: kiali-istio-traffic-generator
+  annotations:
+    openshift.io/display-name: kiali-istio-traffic-generator
+    description: Kiali Traffic Generator for Istio
+
+parameters:
+  - name: IMAGE
+    description: What docker image should be used
+    value: kiali-istio/sws-traffic-generator:1.0
+
+  - name: ROUTE
+    description: What is the ROUTE to be queried
+    value: CHANGEME
```

Comment:
We should remove the 'value' here and make the parameter required. This will force someone to set a value